v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
@Nikerabbit The whole spec is mounted under `/translate` prefix. So the full path will be: `/api/rest_v1/translate/machine/{from}/{to}{/provider}`. I'm not defending the `machine` part though, I don't really like it either, it suggests that not all of the translation is machine-driven